engines/wintermute/ad/ad_scene_geometry.cpp

```diff
@@ -462,6 +462,37 @@ bool AdSceneGeometry::render(bool render) {
 		//		m_Renderer->m_NumPolygons += _generics[i]->m_Mesh->m_NumFaces;
 	}
 
+	_gameRef->_renderer3D->resetModelViewTransform();
+
+	for (i = 0; i < _lights.size(); ++i) {
+		if (!_lights[i]->_active) {
+			continue;
+		}
+
+		glBegin(GL_LINES);
```

Comment:
GL-calls should not happen outside the renderer, otherwise we can't implement multiple renderers easily. Implement what you need inside the renderer, and call from here instead.

Reply:
Discussed this on Discord, will be refactored at a later point.

Reply:
indeed no render code